packages/runtime/src/next/components/tests/controls/rich-text-v2-control/client.test.tsx

```diff
@@ -0,0 +1,23 @@
+/** @jest-environment jsdom */
+import { RichText } from '../../../../../controls'
+
+import { testPageControlPropRendering } from '../page-control-prop-rendering'
+import { value, cacheData } from './fixtures'
+
+test.each([RichText.Mode.Inline, RichText.Mode.Block])(
+  `renders a %s placeholder when empty`,
+  async mode => {
+    await testPageControlPropRendering(RichText({ mode }), {
+      value: undefined,
+      expectedRenders: 1,
+    })
+  },
+)
+
+test(`renders provided text content`, async () => {
+  await testPageControlPropRendering(RichText(), {
+    value,
+    cacheData: cacheData(),
+    expectedRenders: 1,
+  })
+})
```

Comment:
Client tests haven't changed (other than titles), but are quite a bit shorter due to moving fixtures into a separate file.